deploy/gk-deploy

```diff
@@ -110,6 +120,23 @@ Options:
               from the GlusterFS daemonset. This allows for multiple GlusterFS
               daemonsets to run in the same cluster. Default is 'glusterfs'.
 
+  --no-object
+              Don't deploy a gluster-s3 container. Default is to deploy.
+
+  --object-account ACCOUNT, --object-user USER, --object-password PASSWORD
+              Required credentials for deploying the gluster-s3 container. If
+              any of these are missing, object container deployment will be
+              skipped.
+
+  --object-sc STORAGE_CLASS
+              Specify an optional, pre-existing StorageClass for the object
+              container to use to create GlusterFS volumes to host the objects.
+              Default is to create a new StorageClass called '${OBJ_STORAGE_CLASS}'.
```

Comment:
Could we make it a little more clear that the s3 container needs 2 volumes - one for the objects and one for the metadata? Like this it's a little confusing.